src/WorkBC.Tests/Tests/SearchTests.cs

```diff
@@ -203,6 +204,19 @@ public async Task SearchJobsExcludingPlacementAgencyJobs()
             //There should be results
             Assert.True(result.Count > 0, "Jobs excluding agency jobs did not return results");
         }
+        
+        [Theory(DisplayName = "When both salary string and salary numeric fields populated, calculate SalarySummary")]
+        [InlineData(39879152, "N/A")] // below minimum wage
+        [InlineData(40098629, "N/A")] // below minimum wage
+        [InlineData(40104984, "$2,000.00 monthly + 3% commission per sale")]
+        [InlineData(40207035, "$27.50 hourly")]
```

Comment:
` [InlineData(40207035, "$27.50 hourly")]`

Job 40207035 was previously displaying a `null` SalarySummary because of a missing `employer_postal_code` attribute in the XML.